indico/modules/events/editing/client/js/management/editable_type/EditableList.jsx

```diff
@@ -438,30 +453,85 @@ function EditableListDisplay({
       />
     );
   };
+  const renderTags = (__, editable) => {
+    const tags = editable.tags.sort((a, b) => a.code.localeCompare(b.code));
+    const labelTags = () => {
+      return (
+        <div style={{display: 'flex', flexWrap: 'nowrap'}}>
+          {tags.map(t => {
+            return (
```

Comment:
since you only `return` in this arrow function, you can use the `t => (...)` notation (ie remove `return` and `;` and replace the curly braces with parentheses)